SceneMaker/VisualSceneMaker
# Yallahplugin

Check for test material in: `Nextcloud/Shared/ACG\ Material/Projects/Active/MindBot/WP5/T5.8-AvatarVisualizer/201110_VSM\ Integration\ test`

* Install the development environment for VSM (VisualSceneMaker)
  * switch to the `yallahplugin` branch: <https://github.com/SceneMaker/VisualSceneMaker/tree/yallahplugin>
* Open and run project `TestYallah`
* Run a local web server, for example by executing the following: `python -m SimpleHTTPServer 8800`.
* Open the web page: `http://localhost:8800/`
* and browse into `DECAD2-test03`
  * The avatar _Anna_ should start to talk, move eyes, and dance.
